Vagrantfile.containerd2youki

```diff
@@ -0,0 +1,51 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    config.vm.box = "generic/ubuntu2004"
```

Comment:
Can we use ubuntu 22.04?
```suggestion
    config.vm.box = "generic/ubuntu2204"
```

Reply:
https://app.vagrantup.com/generic/boxes/ubuntu2204

Reply:
OK.  Is it ok to set up GitHub Actions based on Ubuntu 22.04 as well?

Reply:
@guni1192 Good question! Yes, please. If you will get errors with Ubuntu 22.04, please tell me and consider using Ubuntu 20.04.